lib/markbind/lib/parser.js

```diff
@@ -85,12 +87,22 @@ Parser.prototype.getMissingIncludeSrc = function () {
   return _.clone(this.missingIncludeSrc);
 };
 
+Parser.prototype._createErrorNode = function (element, error) {
```

Comment:
Can this be [a function instead of a method](https://stackoverflow.com/questions/15285293/method-vs-functions-and-other-questions) since it doesn't use `this`?